Scroller/EditableWrapper.js

```diff
@@ -37,6 +40,12 @@ const EditableShape = PropTypes.shape({
 
 const SpotlightAccelerator = new Accelerator([5, 4]);
 
+const pressDuration = 500;
```

Comment:
```suggestion
const holdDuration= 500;
```
Sorry, I don't like variable name

Reply:
It's fixed.